Makefile

```diff
@@ -10,31 +10,36 @@ ifeq ($(OS),Windows_NT)
 	MV ?= move
 	CP ?= copy /Y
 	RM ?= del /Q /F
-	EXESUFFIX ?= .exe
-	SUFFIX ?= 2>NULL
 else
 	MV ?= mv -f
 	CP ?= cp -f
-	RM ?= rm -rf
-	EXESUFFIX ?=
-	SUFFIX ?=
+	RM ?= rm -f
 endif
 
+$(CXX) ?= g++
+
 all: sass2scss
 
 CXXFLAGS = -DSASS2SCSS_VERSION="\"$(SASS2SCSS_VERSION)\""
```

Comment:
We should add `-std=c++98` to flags as well.

Reply:
Feel free to add PR